rules/google_project_iam_member_invalid_member_format.go

```diff
@@ -0,0 +1,56 @@
+package rules
+
+import (
+	"fmt"
+
+	hcl "github.com/hashicorp/hcl/v2"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+)
+
+// GoogleProjectIamMemberInvalidMemberFormatRule checks whether member value is invalid
+type GoogleProjectIamMemberInvalidMemberFormatRule struct{}
+
+// NewGoogleProjectIamMemberInvalidMemberFormatRule returns new rule with default attributes
+func NewGoogleProjectIamMemberInvalidMemberFormatRule() *GoogleProjectIamMemberInvalidMemberFormatRule {
+	return &GoogleProjectIamMemberInvalidMemberFormatRule{}
+}
+
+// Name returns the rule name
+func (r *GoogleProjectIamMemberInvalidMemberFormatRule) Name() string {
+	return "google_project_iam_member_invalid_member_format"
+}
+
+// Enabled returns whether the rule is enabled by default
+func (r *GoogleProjectIamMemberInvalidMemberFormatRule) Enabled() bool {
+	return true
+}
+
+// Severity returns the rule severity
+func (r *GoogleProjectIamMemberInvalidMemberFormatRule) Severity() string {
+	return tflint.ERROR
+}
+
+// Link returns the rule reference link
+func (r *GoogleProjectIamMemberInvalidMemberFormatRule) Link() string {
+	return iamMemberFormatDocumentLink
```

Comment:
This should return a link to the documentation in this repository.

```suggestion
	return project.ReferenceLink(r.Name())
```

Reply:
Fixed https://github.com/terraform-linters/tflint-ruleset-google/pull/121/commits/783a2b81a41faa9e3d7c80f647d966cca46a4e14